java/arcs/core/data/Plan.kt

```diff
@@ -97,6 +90,8 @@ open class Plan(
     override fun hashCode(): Int = particles.hashCode()
 
     companion object {
-        val particleLens = lens(Plan::particles) { t, f -> Plan(particles = f, arcId = t.arcId) }
+        val particleLens = lens(Plan::particles) {
+            t, f -> Plan(particles = f, annotations = t.annotations)
```

Comment:
Nit: `t, f ->` goes on the previous line, `Plan(...)` stays on this line.

Also, are there better names you could use for `t, f`?

Reply:
rearranged lines.
the `t, f` are used everywhere with the Lens utility (looking at Lens.kt i think they stand for target and focus). keeping them as is for now to be consistent with the rest of the codebase.

Reply:
They're adopted Haskellisms. You can call them target, focus if you want.